docs/_snippets/framework/extending-content-custom-figure-attributes.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals ClassicEditor, console, window, document */
+
+import { CS_CONFIG } from '@ckeditor/ckeditor5-cloud-services/tests/_utils/cloud-services-config';
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
```

Comment:
Same here – you can't add plugins like this.

Reply:
Fixed.